book/src/csi-driver-object.md

```diff
@@ -38,58 +38,46 @@ There are three important fields:
+    * `csi.storage.k8s.io/pod.namespace": pod.Namespace`
+    * `csi.storage.k8s.io/pod.uid": string(pod.UID)`
   * For more information see [Pod Info on Mount](pod-info.md).
 
 ## What creates the CSIDriver object?
 
 CSI drivers do not need to create the `CSIDriver` object directly. Instead they may use the [cluster-driver-registrar](cluster-driver-registrar.md) sidecar container (customizing it as needed with startup parameters) -- when deployed with a CSI driver it automatically creates a `CSIDriver` CR representing the driver.
 
-### Enabling CSIDriver
-
-The `CSIDriver` object is available as alpha starting with Kubernetes v1.12. Because it is an alpha feature, it is disabled by default.
-It is planned to be moved to beta in Kubernetes v1.14 and enabled by default.
-
-To enable the use of `CSIDriver` on Kubernetes, do the following:
-
-1) Ensure the feature gate is enabled via the following Kubernetes feature flag: `--feature-gates=CSIDriverRegistry=true`
-2) Either ensure the `CSIDriver` CRD is automatically installed via the [Kubernetes Storage CRD addon](https://github.com/kubernetes/kubernetes/tree/master/cluster/addons/storage-crds) OR manually install the `CSIDriver` CRD on the Kubernetes cluster with the following command:
-```
-$> kubectl create -f https://raw.githubusercontent.com/kubernetes/csi-api/master/pkg/crd/manifests/csidriver.yaml
-```
-
 ### Listing registered CSI drivers
-Using the `CSIDriver` CRD, it is now possible to query Kubernetes to get a list of registered drivers running in the cluster as shown below:
+Using the `CSIDriver` object, it is now possible to query Kubernetes to get a list of registered drivers running in the cluster as shown below:
 
 ```
-$> kubectl get csidrivers.csi.storage.k8s.io
+$> kubectl get csidrivers.storage.k8s.io
 NAME           AGE
 csi-hostpath   2m
 ```
 Or get a more detailed view of your registered driver with:
 ```
-$> kubectl describe csidrivers.csi.storage.k8s.io
+$> kubectl describe csidrivers.storage.k8s.io
 Name:         csi-hostpath
 Namespace:
 Labels:       <none>
 Annotations:  <none>
-API Version:  csi.storage.k8s.io/v1alpha1
+API Version:  storage.k8s.io/v1beta1
 Kind:         CSIDriver
 Metadata:
   Creation Timestamp:  2018-10-04T21:15:30Z
   Generation:          1
   Resource Version:    390
-  Self Link:           /apis/csi.storage.k8s.io/v1alpha1/csidrivers/csi-hostpath
+  Self Link:           /apis/storage.k8s.io/v1beta1/csidrivers/csi-hostpath
   UID:                 9f854aa6-c81a-11e8-bdce-000c29e88ff1
 Spec:
   Attach Required:            true
-  Pod Info On Mount Version:
+  Pod Info On Mount:          false
 Events:                       <none>
 ```
 
+## Changes from Kubernetes 1.13
+
+During alpha development, `CSIDriver` was CRD in `csi.storage.k8s.io/v1alpha1` API group. This API is no longer supported and new beta `CSIDriver` in `storage.k8s.io/v1beta1` must be used. There is no automatic update of existing CRDs and their CRs during Kubernetes update.
```

Comment:
Let's change this section:

## Changes from Alpha to Beta

### CRD to Built in Type

During alpha development, the `CSIDriver` object was also defined as a [Custom Resource Definition](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/#create-a-customresourcedefinition) (CRD). As part of the promotion to beta the object has been moved to the built-in Kubernetes API.

In the move from alpha to beta, the API Group for this object changed from `csi.storage.k8s.io/v1alpha1` to `storage.k8s.io/v1beta1`.

There is no automatic update of existing CRDs and their CRs during Kubernetes update to the new build-in type.

### Enabling CSIDriver on Kubernetes

In Kubernetes v1.12 and v1.13, because the feature was alpha, it was disabled by default. To enable the use of `CSIDriver` on these versions, do the following:

1) Ensure the feature gate is enabled via the following Kubernetes feature flag: `--feature-gates=CSIDriverRegistry=true`	
2) Either ensure the `CSIDriver` CRD is automatically installed via the [Kubernetes Storage CRD addon](https://github.com/kubernetes/kubernetes/tree/master/cluster/addons/storage-crds) OR manually install the `CSIDriver` CRD on the Kubernetes cluster with the following command:	
```	
$> kubectl create -f https://raw.githubusercontent.com/kubernetes/csi-api/master/pkg/crd/manifests/csidriver.yaml	
```	

Kubernetes v1.14+, uses the same Kubernetes feature flag, but because the feature is beta, it is enabled by default. And since the API type (as of beta) is built in to the Kubernetes API, installation of the CRD is no longer required.